celery_worker/tests/test_tasks.py

```
@@ -0,0 +1,61 @@
+
+
+class TestProcessDrawing:
+    """Тесты основной задачи process_drawing"""
+
+    @patch("worker.requests.post")
+    @patch("worker.get_drawing_sync")
+    def test_process_drawing_success(self, mock_get_drawing, mock_post):
+        """Успешный сценарий: агент отвечает → задача завершается"""
+        mock_get_drawing.return_value = {
+            "id": "123",
+            "file_path": "/tmp/test.png",
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/test.png"

(S108)